Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -3906,6 +3905,10 @@ struct RenderBox::PositionedLayoutConstraints {
     const CheckedPtr<const RenderBoxModelObject> defaultAnchorBox; // Only set if needed.
     LayoutRange anchorArea; // Only valid if defaultAnchor exists.
 
+    /* These values are cached by the constructor, and should not change afterwards. */
+    LayoutUnit bordersPlusPadding;
+    bool useStaticPosition;
```

Comment:
It is good idea to initialize fields without constructors. It is easy to change code later and end up with a path that leaves the field uninitialized.
```
bool useStaticPosition { false };
```